engine/flink/avro-components-utils/src/test/scala/pl/touk/nussknacker/engine/avro/AvroSchemaSpelExpressionSpec.scala

```diff
@@ -138,26 +138,16 @@ class AvroSchemaSpelExpressionSpec extends FunSpec with Matchers {
 
   it("should recognize record with enum") {
     val ctx = ValidationContext.empty.withVariable("input", AvroSchemaTypeDefinitionExtractor.typeDefinition(PaymentV1.schema), paramName = None).toOption.get
-
-    parse[CharSequence]("#input.currency", ctx) should be ('valid)
```

Comment:
Why it is out?

Reply:
EnumSymbol isn't String.

Reply:
if you want it so badly, I can rollback schema to previous version and do something like this:

`parse[CharSequence]("#input.currency.toString", ctx) should be ('valid)` 

but does it have sens?

Reply:
Yes it was the point :) Why it doesn't have sense?

Reply:
In same way you can compare int / other objects.. .toString always will return string?